src/data/panel.ts

```diff
@@ -44,3 +44,39 @@ export const getPanelTitle = (hass: HomeAssistant): string | undefined => {
 
   return hass.localize(`panel.${panel.title}`) || panel.title || undefined;
 };
+
+export const getPanelText = (hass: HomeAssistant, panel: Panels["panel"]) => {
+  let translationKey = "";
+
+  if (!panel.title) {
+    switch (panel.component_name) {
+      case "profile":
+        translationKey = "panel.profile";
+        break;
+      case "lovelace":
+        translationKey = "panel.states";
+        break;
+    }
+  } else {
+    translationKey = `panel.${panel.title}`;
+  }
+
+  return hass.localize(
+    "ui.dialogs.quick-bar.commands.navigation.navigate_to",
```

Comment:
This piece belongs in the quick bar code.

Reply:
OOPS

Reply:
Done